learn/what_is_meilisearch/language.md

```diff
@@ -1,42 +1,43 @@
 # Language
 
-**Meilisearch is multilingual**, featuring optimized support for:
+Meilisearch is multilingual, featuring optimized support for:
 
-- **Any language that uses whitespace to separate words**
-- **Chinese** (through [Jieba](https://github.com/messense/jieba-rs))
-- **Japanese** (through [Lindera](https://github.com/lindera-morphology/lindera))
+- Any language that uses whitespace to separate words
+- Chinese
+- Japanese
+- Hebrew
 
-We aim to provide global language support, and your feedback helps us [move closer to that goal](#improving-our-language-support). If you notice inconsistencies in your search results or the way your documents are processed, please open an issue on our [GitHub repository](https://github.com/meilisearch/meilisearch/issues/new/choose).
+We aim to provide global language support, and your feedback helps us move closer to that goal. If you notice inconsistencies in your search results or the way your documents are processed, please [open an issue in our tokenizer repo](https://github.com/meilisearch/charabia/issues/new).
```

Comment:
It's not so easy for the users to know if the tokenizer is the source of the issue, I'd prefer that users create an issue on Meilisearch in order to be moved or copied in the good repository by a core-team member. 😊

```suggestion
We aim to provide global language support, and your feedback helps us move closer to that goal. If you notice inconsistencies in your search results or the way your documents are processed, please [open an issue in Meilisearch repo](https://github.com/meilisearch/meilisearch/issues/new/choose).
```

Reply:
Basically, it's better to redirect users to [Meilisearch issues creation page](https://github.com/meilisearch/meilisearch/issues/new/choose) when they want to create an issue.

The only exception I see is the Language support  where we redirect users:
https://github.com/meilisearch/product/discussions?discussions_q=label%3Aproduct%3Acore%3Atokenizer

Maybe it's a good idea to add a specific redirection in the Meilisearch issues creation page, what do you think @curquiza?

Reply:
Hello! I agree with many: any issue when using Meilisearch should be in the Meilisearch repo. The users cannot really guess if it's related to the tokenizer or not, and it's not their job.
The exception is indeed for the language support request we expect on the product repo -> we've just merged [a PR](https://github.com/meilisearch/meilisearch/pull/2592) to update our issue templates

So, maybe what you want to do (docs team) is only to encourage people to open a language request on the product side.